tests/lib/rules/avoid-leaking-state-in-ember-objects.js

```diff
@@ -44,6 +44,40 @@ eslintTester.run('avoid-leaking-state-in-components', rule, {
       code: 'export default Component.extend({ classNames: [], classNameBindings: [], actions: {}, concatenatedProperties: [], mergedProperties: [], positionalParams: [] });',
       parserOptions: { ecmaVersion: 6, sourceType: 'module' },
     },
+
+    {
+      code: 'export default Foo.extend();',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    },
+    {
+      code: 'export default Foo.extend({ someProp: "example", init() { this.set("anotherProp", []) } });',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    },
+    {
+      code: 'export default Foo.extend({ someProp: "example", init() { this.set("anotherProp", {}) } });',
+      parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+    },
+    {
+      code: 'export default Foo.extend({ someProp: "example", init() { this.set("anotherProp", new Ember.A()) } });',
```

Comment:
you should use template literals so you can format the code on multiple lines and make it waaay easier to read. [Like this](https://github.com/ember-best-practices/eslint-plugin-ember-best-practices/blob/master/tests/lib/rules/no-attrs.js#L14)